### PR TITLE
feat: Windows WSL Support

### PR DESCRIPTION
**Pull request by zfields**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,21 +102,16 @@ If you cloned this repository directly to start a new project this file does
...
+be named based on what you call your project in the top-level `CMakeLists.txt`
+file.
 
 ### tasks.json
 
 No changes need to be made to this file.
 
 ## Debugging Instructions
 
-> _**NOTE:** One thing I encountered was both the ESP32 target device as well as
```
Comment (zfields) [inline README.md:113]: migrated to **Troubleshooting* section

File: README.md [hunk 2]
```diff
@@ -101,21 +102,16 @@ If you cloned this repository directly to start a new project this file does
...
-> the ESP-Prog will enumerate as `/dev/ttyUSB0` if plugged in first on Linux.
-> Since the debug commands rely on the target device port being constant it's
-> important that you plug in the target device first if you've configured
-> **settings.json** to use `/dev/ttyUSB0` as the flash and monitor port._
-
```
Comment (zfields) [inline README.md:118]: migrated to **Troubleshooting** section

File: README.md [hunk 3]
```diff
@@ -61,7 +62,7 @@ git clone --recursive https://github.com/espressif/esp-idf.git -b v5.0
 ```
 
 > _**NOTE:** If branch does not exist yet, then simply use `master` (v5.0 and
```
Comment (sidwarkd) [inline README.md:64]:
```suggestion
> _**NOTE:** If branch does not exist yet, then simply use `main` (v5.0 and
```

**Review by sidwarkd — APPROVED** [review]
We have `main` instead of `master`. Just noticed it going through so added a suggested change. Otherwise, looks good.

**Comment by zfields** [comment]
`espressif/esp-idf` uses `master`

**Pull request merged**